src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -29,9 +29,7 @@ public class HubEndPoint<THub> : HubEndPoint<THub, IClientProxy> where THub : Hu
 
     public class HubEndPoint<THub, TClient> : EndPoint, IInvocationBinder where THub : Hub<TClient>
     {
-        private readonly Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>> _callbacks
-            = new Dictionary<string, Func<Connection, InvocationDescriptor, Task<InvocationResultDescriptor>>>(StringComparer.OrdinalIgnoreCase);
-        private readonly Dictionary<string, Type[]> _paramTypes = new Dictionary<string, Type[]>();
+        private readonly Dictionary<string, HubMethodDescriptor> _methods = new Dictionary<string, HubMethodDescriptor>(StringComparer.OrdinalIgnoreCase);
```

Comment:
I would like it to be case insensitive, but if we care about correctness if would need to be case sensitive. e.g. Hub with 2 methods `Echo` and `echo`